libs/cdk/src/lib/utils/helpers/get-native-element.ts

```diff
@@ -19,6 +19,6 @@ export function getNativeElement<T extends Element = HTMLElement>(
  */
 export function isHasElementRef<T extends Element = HTMLElement>(
```

Comment:
Maybe when we are refactoring we can renaming this confusing function too. **isHas** and just leave **has**